cmd/influx/setup.go

```diff
@@ -81,12 +97,24 @@ func setupF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to setup instance: %v", err)
 	}
 
-	err = writeTokenToPath(result.Auth.Token, dPath, dir)
+	if len(existingConfigs) > 0 {
+		existingConfigs[setupFlags.name] = config.Config{
+			Token: result.Auth.Token,
+			Org:   result.Org.Name,
+			Host:  flags.Host,
+		}
+		config.LocalConfigsSVC{
+			Path: dPath,
+			Dir:  dir,
+		}.WriteConfigs(existingConfigs)
+	} else {
+		err = writeConfigToPath(result.Auth.Token, result.Org.Name, dPath, dir)
```

Comment:
why isn't hte LocolConfigSVC called here? can it be instead? feels like we should encapsulate this into the service